core/domain/entities/editor/blocks/EventAttendees.php

```diff
@@ -63,34 +63,35 @@ public function initialize()
                 'EventEspresso\core\domain\entities\route_match\specifications\admin\EspressoStandardPostTypeEditor',
                 'EventEspresso\core\domain\entities\route_match\specifications\admin\WordPressPostTypeEditor',
                 'EventEspresso\core\domain\entities\route_match\specifications\frontend\EspressoBlockRenderer',
+                'EventEspresso\core\domain\entities\route_match\specifications\frontend\AnyFrontendRequest'
```

Comment:
I'm not sure how the EventAttendee block _ever_ rendered on the frontend because without this it wouldn't (because this block class wouldn't load).

Reply:
because it was using SSR and that callback is saved with the block info, so when you hit the frontend, it only needs to hit that callback and not actually load the block?

it WAS loading for me as well as any one that tested that pull.

did you HAVE to add this to get it to work?

Reply:
> because it was using SSR and that callback is saved with the block info,

Even if the callback is saved with the block info, if the class containing the rendered is not loaded, then the callback can't be called right?  That's what was happening in my case and why I explicitly had to add this code.  The renderer function was not working.  At first I thought maybe the `EspressoBlockRenderer` route matcher wasn't working as expected but as far as I could tell, its _intentionally_ only listening for the SSR endpoint call.  Maybe at some point that route matcher was looser and that's why things were working in testing against the frontend?

Reply:
I'm now seeing that the route matchers are only used for determining whether the block should be registered and for whether assets should be registered.  So I'm not sure now why it wasn't getting rendered for me, I'll revisit.

Reply:
So I revisited and verified the block content will _not_ render unless I have the new route matcher in place (verified by doing a `var_dump` in the block `render()` method).  I'm guessing what might have changed between you implementing this and now is that WP now longer only renders content for _registered_ blocks.  It's probably a failsafe for when a plugin is deactivated because wp won't know what to do.  It also makes sense because block registration involves the ability to register frontend scripts separate from backend scripts.  So if WP rendered the block without registration then any scripts/styles would not get loaded.